buildSrc/src/main/kotlin/pg-index-health.java-conventions.gradle.kts

```diff
@@ -36,7 +37,7 @@ tasks.withType<JavaCompile>().configureEach {
 
 tasks {
     test {
-        dependsOn(checkstyleMain, checkstyleTest, pmdMain, pmdTest, spotbugsMain, spotbugsTest)
+        dependsOn(checkstyleMain, checkstyleTest, pmdMain, pmdTest, spotbugsMain, spotbugsTest, forbiddenApisMain, forbiddenApisTest)
```

Comment:
Move `(forbiddenApisMain, forbiddenApisTest)` to the buildSrc/src/main/kotlin/pg-index-health.forbidden-apis.gradle.kts